runtime/development/src/evm/precompile.rs

```diff
@@ -0,0 +1,46 @@
+use core::marker::PhantomData;
+
+use codec::Decode;
+use frame_support::dispatch::{Dispatchable, GetDispatchInfo, PostDispatchInfo};
+use pallet_evm::{Precompile, PrecompileHandle, PrecompileResult, PrecompileSet};
+use sp_core::H160;
+
+pub struct CentrifugePrecompiles<R>(PhantomData<R>);
+
+impl<R> CentrifugePrecompiles<R> {
+	#[allow(clippy::new_without_default)] // We'll never use Default and can't derive it.
+	pub fn new() -> Self {
+		Self(Default::default())
+	}
+}
+
+// 1025 is chosen for compatibility with Moonbeam
+const DISPATCH_ADDR: H160 = addr(1025);
```

Comment:
Can you explain why the value 1025 grants that compatibility?

Reply:
Because it's the value that Moonbeam chose. As far as I know it was arbitrary on their end. But it made sense to me to have common precompiles have the same address within the polkadot ecosystem.